releasenotes/notes/add-sampler-qnn-a093431afc1c5441.yaml

```diff
@@ -0,0 +1,54 @@
+---
+features:
+  - |
+    Introduced Sampler Quantum Neural Network
+    (:class:`~qiskit_machine_learning.neural_networks.SamplerQNN`) based on (runtime) primitives.
+    This implementation leverages the sampler primitive
+    (see :class:`~qiskit.primitives.BaseSampler`) and the sampler gradients
+    (see :class:`~qiskit.algorithms.gradients.BaseSamplerGradient`) to enable runtime access and
+    more efficient computation of forward and backward passes more efficiently.
+
+    The new Sampler QNN exposes a similar interface to the Circuit QNN, with a few differences.
```

Comment:
Is there a reason not to have SamplerQNN and CircuitQNN  instead of having spaces in there. The former will match a search the other not so much. I see it was done as Oflow QNN with a space in the Estimator release note though EstimatorQNN I see just as the one word.

Reply:
I don't see a reason. Updated with the links.
I can separately update the EstimatorQNN reno if required. Let me know.